helpers/helpers.go

```diff
@@ -55,6 +55,44 @@ func ExpiryTime(chain []*x509.Certificate) *time.Time {
 	return &notAfter
 }
 
+// MonthsValid returns the number of months for which a certificate is valid.
+func MonthsValid(c *x509.Certificate) int {
+	issued := c.NotBefore
+	expiry := c.NotAfter
+	years := (expiry.Year() - issued.Year())
+	months := years*12 + int(expiry.Month()) - int(issued.Month())
+
+	// Round up if valid for less than a full month
+	if expiry.Day() > issued.Day() {
+		months++
+	}
+	return months
+}
+
+// ValidExpiry determines if a certificate is valid for an acceptable
+// length of time per the CA/Browser Forum baseline requirements.
+// See https://cabforum.org/wp-content/uploads/CAB-Forum-BR-1.3.0.pdf
+func ValidExpiry(c *x509.Certificate) bool {
+	issued := c.NotBefore
+	jul2012 := time.Date(2012, time.July, 01, 0, 0, 0, 0, time.UTC)
```

Comment:
If these are moved outside the function, the function won't have to compute the values each time.